modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach error issues
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node. However, this is not possible because when the node is down, the system cannot unmount the volume. A multi-attach error message is reported, for example:
+
+[source,terminal]
+.Example output
+--
+Unable to attach or mount volumes: unmounted volumes=[sso-mysql-pvol], unattached volumes=[sso-mysql-pvol default-token-x4rzc]: timed out waiting for the condition
+Multi-Attach error for volume "pvc-8837384d-69d7-40b2-b2e6-5df86943eef9" Volume is already used by pod(s) sso-mysql-1-ns6b4
+--
+
+.Solution
+.. *Use RWX volumes.*
++
+For most storage solutions, the solution to the multi-attach error is to use ReadWriteMany (RWX) volumes.
```

Comment:
It might be worth changing 'storage solutions' to something like 'storage types' or 'storage platforms', to avoid using the word 'solution' twice in quick succession.

Reply:
Also, I would say 'For most storage solutions, you can use ReadWriteMany (RWX) volumes to prevent multi-attach errors'. I.e. it is more of a prevention than a solution to the error itself. Although, I'm not sure what is involved in converting an RWO volume to an RWX volume after the error has occurred.

Reply:
I've just realised that my second point here negates the first :-)

Reply:
I agree with your suggestion that it is more of a preventive measure and I've changed the wording accordingly. I'm not sure what's involved in converting, either.